src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationTrailingSpace.java

```diff
@@ -0,0 +1,32 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config:  default
+ * location="SECOND_LINE"
+ */
+public final class InputJavadocSummaryLocationTrailingSpace {
+
+    /**  
+     * ^^ There is a trailing space in the first line. // OK
+     */
+    private int space;
+
+    /**********************
+     * Trailing asterisks in the first line * // OK
+     *
+     **********************/
+    private int asterisks;
+
+    /** ******
+     * There is a space, then leading asterisks in the first line. // OK
```

Comment:
if want to be close to javadoc tool in behavior, that ignore all trailing `*` and spaces ....
We need to make some Note to xdoc, to let user aware, we can not expect users know all such nuances.

Reply:
Done. Added a note about leading asterisks on the first line.